src/foundation.cs

```diff
@@ -8194,15 +8194,23 @@ interface NSInputStream {
 
 	}
 
-	delegate bool NSEnumerateLinguisticTagsEnumerator (NSLinguisticTagScheme tag, NSRange tokenRange, NSRange sentenceRange, ref bool stop);
+	delegate bool NSEnumerateLinguisticTagsEnumerator (NSString tag, NSRange tokenRange, NSRange sentenceRange, ref bool stop);
 
 	[Category]
 	[BaseType (typeof(NSString))]
 	interface NSLinguisticAnalysis {
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
 		[Export ("linguisticTagsInRange:scheme:options:orthography:tokenRanges:")]
+		string[] GetLinguisticTags (NSRange range, NSString scheme, NSLinguisticTaggerOptions options, [NullAllowed] NSOrthography orthography, [NullAllowed] out NSValue[] tokenRanges);
```

Comment:
missed that earlier - but the tests made it more obvious... (even if that one is **not** tested)

it seems to be returning `NSLinguisticTag*` ?
because then would be best to return `NSLinguisticTag[]` in that case
or _maybe_ `NSString` but not `string`